test/integration/conftest.py

```diff
@@ -11,12 +11,28 @@
 import pytest
 
 import pyexasol
+from noxconfig import CONTAINER_NAME
```

Comment:
Ah, hm, I didn't think about this dependency when I made the suggestion.
@tkilias , do you think this is problematic?

Reply:
I could alternately add a fixture for it with the fixed text. atm, it is also hard-coded separately for the `DockerDataLoader` later in the `conftest.py`.

Reply:
hm, importing from the noxconfig, it probably can cause issues and make the test potentially depend on your current working directory

Reply:
@ArBridgeman yes, the fixture is safer